plugins/analytics-module-matomo/README.md

```diff
@@ -40,20 +40,20 @@ export const apis: AnyApiFactory[] = [
 3. Configure the plugin in your `app-config.yaml`:
 
 The following is the minimum configuration required to start sending analytics
-events to GA. All that's needed is your Universal Analytics tracking ID:
+events to Matomo. All that's needed is your Matomo Host and Site Id:
 
 ```yaml
 # app-config.yaml
 app:
   analytics:
     matomo:
-      instanceUrl: ${ANALYTICS_MATOMO_INSTANCE_URL}
+      host: ${ANALYTICS_MATOMO_INSTANCE_URL}
```

Comment:
I would change the variable name to ANALYTICS_MATOMO_HOST. It threw me off. I was putting the entire URL, including https:// and it didn't work.

Reply:
Try using it under App

```
app:
  title: Backstage
  analytics:
    matomo:
      instanceUrl: ${ANALYTICS_MATOMO_INSTANCE_URL}
      siteId: ${ANALYTICS_MATOMO_SITE_ID}
``` this should work

Reply:
I rechecked the configs its working for us

Reply:
Did you added site id.


Our config look like this ``````
app:
  title: Backstage
  analytics:
    matomo:
      instanceUrl: ${ANALYTICS_MATOMO_INSTANCE_URL}
      siteId: ${ANALYTICS_MATOMO_SITE_ID}
``````

Reply:
for making it as host vcariable it needs to be changed in the code first

Reply:
I didn't want to change the Input Variable since it will be breaking for the ones that already use the plugin.